extensions/wikia/AdEngine/js/provider/netzathleten.js

```diff
@@ -0,0 +1,73 @@
+/*global define, require*/
+define('ext.wikia.adEngine.provider.netzathleten', [
+	'ext.wikia.adEngine.slotTweaker',
+	'ext.wikia.adEngine.utils.eventDispatcher',
+	'wikia.document',
+	'wikia.lazyqueue',
+	'wikia.window'
+], function (slotTweaker, eventDispatcher, doc, lazyQueue, win) {
+	'use strict';
+
+	var initialized = false,
+		libraryUrl = '//s.adadapter.netzathleten-media.de/API-1.0/NA-828433-1/naMediaAd.js',
+		cmdQueue = [],
+		slotMap = {
+			TOP_LEADERBOARD: 'SUPERBANNER',
+			TOP_RIGHT_BOXAD: 'MEDIUM_RECTANGLE',
+			PREFOOTER_LEFT_BOXAD: 'MEDIUM_RECTANGLE',
+			MOBILE_TOP_LEADERBOARD: 'TOP',
+			MOBILE_IN_CONTENT: 'MID',
+			MOBILE_PREFOOTER: 'BOTTOM'
+		};
+
+	function loadLibrary() {
+		var script = doc.createElement('script');
+
+		script.src = libraryUrl;
+		script.async = true;
+		script.onload = function () {
+			cmdQueue.start();
+		};
+
+		doc.head.appendChild(script);
+	}
+
+	function init() {
+		lazyQueue.makeQueue(cmdQueue, function (callback) {
+			callback();
+		});
+
+		loadLibrary();
+
+		eventDispatcher.dispatch('wikia.not_uap');
+		cmdQueue.push(function () {
+			win.naMediaAd.setValue('homesite', false);
+		});
+		initialized = true;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "initialized" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS1854)